dev_readme.md

```diff
@@ -37,7 +85,18 @@
 * Root level: this is the level of the project which contains all the others as files and subfolders
 * Terminal: this is an application on your computer which allows you to use code to modify your computer's software and files. 
 
-## Extra Tips
+## How-Tos 
+
+### Cloning the Repository
```

Comment:
Really useful to have this here - maybe we could do an internal link to it in the instructions for people who are unsure how to follow this step so it's even easier?

Reply:
An internal link? In the `dev_readme.md` or in the `readme.md` file?

Reply:
I was thinking in the dev_readme.md since the first step is cloning?

Reply:
So, like. you want to  embed a link to this section where git cloning is first mentioned?

Reply:
Yeah, exactly!

Reply:
Done. Verify please~

Reply:
Yep! Perfect! Verified!